sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestHelper.scala

```diff
@@ -47,6 +47,7 @@ trait SQLQueryTestHelper {
       .replaceAll("Last Access.*", s"Last Access $notIncludedMsg")
       .replaceAll("Partition Statistics\t\\d+", s"Partition Statistics\t$notIncludedMsg")
       .replaceAll("\\*\\(\\d+\\) ", "*") // remove the WholeStageCodegen codegenStageIds
+      .replaceAll("@[0-9a-z]+,", "@x,") // remove hashCode
```

Comment:
Add `,` to avoid `userinfo@spark.apache.org` being replaced by mistake

https://github.com/apache/spark/blob/9b647e80b9e6b9a2b7afc9045cb8c4aa9b690aae/sql/core/src/test/resources/sql-tests/results/url-functions.sql.out#L55

Reply:
cc @cloud-fan @dongjoon-hyun @ulysses-you 

Change to add a new `replaceAll` action to `SQLQueryTestHelper#replaceNotIncludedMsg` to replace `@hashCode` to `@x`.

Reply:
this one change to remove all